README.md

```diff
@@ -96,6 +97,17 @@ export default {
 
 Alternatively, you can use `openapi-ts.config.js` and configure the export statement depending on your project setup.
 
+### Clients
+
+We provide a variety of possible clients to use when generating your `openapi-ts` client. The following are available:
+
+- `angular`: An [Angular](https://angular.io/) client using [RxJS](https://rxjs.dev/).
```

Comment:
I'm not familiar with Angular, does the generated client work with all versions? Specific range?

Reply:
I am not sure either. We will need to look into this further to say for sure.